R/mergedatasetsbycase.R

```diff
@@ -2515,6 +2515,7 @@ combineAsTextVariable <- function(var.list, data.sets, v.types)
 #' @noRd
 parseTextVariable <- function(text.variable, parser)
 {
+    text.variable <- as.character(text.variable)
```

Comment:
Deserves a comment explaining why `as.character` is needed, or even better would be to sanitise the data further upstream just after the data is read from the file, so that `as.character` isn't needed in a few places.